checks.d/process.py

```diff
@@ -328,6 +329,11 @@ def check(self, instance):
             # we use Process(pid) as a means to search, if pid not found
             # psutil.NoSuchProcess is raised.
             pids = set([psutil.Process(pid).pid])
+        elif pid_file is not None:
+            with open(pid_file, 'r') as file_pid:
+                pid_line = file_pid.readline().strip()
+                pids = set([psutil.Process(int(pid_line[-1])).pid])
```

Comment:
`pid_line` contains the pid as a string, so you shouldn't only select the last character of the string with `pid_line[-1]`, instead this should simply read:
```python
pids = set([psutil.Process(int(pid_line)).pid])
```

To demonstrate this, you could update your test fixture so that it contains a pid that has more than 1 digit